playgrounds/angular/src/app/app.component.ts

```diff
@@ -30,6 +29,7 @@ import {
 import {
   DxPopoverComponent
 } from 'devextreme-angular';
+import ArrayStore from 'devextreme/artifacts/npm/devextreme/data/array_store';
```

Comment:
This import looks strange. Can't we replace it with other one?

Reply:
I agree. Fixed.